ansible_ai_connect/users/pipeline.py

```diff
@@ -84,6 +84,17 @@ def github_get_username(strategy, details, backend, user=None, *args, **kwargs):
         return get_username(strategy, details, backend, user, *args, **kwargs)
 
 
+def is_rh_email_domain(email):
+    email_domain: str = email[email.find("@") + 1 :] if email else None
+    return email_domain.lower() == "redhat.com" if email_domain else False
+
+
+def is_rh_internal_user(user):
+    return is_rh_email_domain(user.email) and (
+        user.email_verified if user.email_verified is not None else False
```

Comment:
`user.email_verified if user.email_verified is not None else False` is equivalent to simply `user.email_verified` (as far as `user.email_verified` field exists

Reply:
yeahh, but not sure if can we guarantee it will always exist...

Reply:
this `if user.email_verified` already step on it

Reply:
if the field is part of the object already, then you can simply check `if user.email_verified`. 

In python,  `user.email_verified is None` is equivalent to `user.email_verified == False`